File: bricks/pac_native_plugin/__brick__/packages/{{name.snakeCase()}}/ios/{{name.snakeCase()}}/Sources/{{name.snakeCase()}}/Domain/Model/{{name.pascalCase()}}Data.swift
// Copyright (c) 2026, one of DanhDue ExOICTIF projects. All rights reserved.

import Foundation

/// Domain model representing data handled by the plugin.
public struct {{name.pascalCase()}}Data: Equatable, Sendable {
    public let id: String
    public let title: String
    public let timestamp: Date

    public init(id: String, title: String, timestamp: Date = Date()) {
        self.id = id
        self.title = title
        self.timestamp = timestamp
    }
}

File: bricks/pac_native_plugin/__brick__/packages/{{name.snakeCase()}}/android/src/main/kotlin/com/danhdue/{{name.snakeCase()}}/domain/model/{{name.pascalCase()}}Data.kt
/*
 * Copyright (c) 2026, one of DanhDue ExOICTIF projects. All rights reserved.
 */

package com.danhdue.{{name.snakeCase()}}.domain.model

data class {{name.pascalCase()}}Data(
    val id: String,
    val title: String,
    val timestamp: Long = System.currentTimeMillis()
)
